cromwell-helm/templates/batch-analysis-ui.yaml

```diff
@@ -0,0 +1,25 @@
+###########################
+# Deployment
+###########################
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "app.fullname" . }}-batch-analysis-ui-depl
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "app.batchAnalysisUI.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "app.batchAnalysisUI.selectorLabels" . | nindent 8 }}
+    spec:
+      containers:
+        - name: {{ .Values.batchAnalysisUI.name }}-container
+          image: us.gcr.io/broad-dsp-gcr-public/terra-batch-analysis-ui:hello-world-ui-2
```

Comment:
`hello-world-ui-2`?

Reply:
As part of publishing the GCR image, I had tagged the UI with `hello-world-ui-2` instead of assigning specific version as it was just a hello world UI. But once we start building CBAS-UI this should get replaced with actual versions.